src/ucp/core/ucp_mm.c

```diff
@@ -1177,22 +1257,22 @@ void ucp_mem_rcache_cleanup(ucp_context_h context)
 
 ucs_status_t ucp_mem_reg_md_map_update(ucp_context_h context)
 {
-    ucp_mem_dummy_handle_t memh = {
-        .memh.alloc_md_index = UCP_NULL_RESOURCE,
-    };
+    ucp_mem_dummy_handle_t memh = {};
```

Comment:
why removed initialization? `UCP_NULL_RESOURCE` is not `0`

Reply:
it will be needed for future changes - see the PR which introduces import/export - #8552